.travis.yml

```diff
@@ -65,17 +65,17 @@ jobs:
     - name: Verify manifest lists
       stage: build-manifest
       script: make build-manifest
-    #- name: Minikube test on amd64
-    #  stage: minikube-e2e-test
-    #  os: linux
-    #  arch: amd64
-    #  script: travis_wait 45 make setup minikube-test-e2e || travis_terminate 1
     # Build all non-ignored releases
     - name: Build image on amd64 and test
       stage: rebuild-e2e-test
       os: linux
       arch: amd64
       script: travis_wait 45 make setup test-e2e RELEASE_TARGET="releases" || travis_terminate 1
+    - name: Minikube test on amd64
```

Comment:
We should run the minikube test right after e2e-test so if there is any issue with release images, the image build and push will be prevented.